docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,146 @@
+#
+#  Copyright 2018-2019 Expedia Group, Inc.
+#
+#       Licensed under the Apache License, Version 2.0 (the "License");
+#       you may not use this file except in compliance with the License.
+#      You may obtain a copy of the License at
+#
+#           http://www.apache.org/licenses/LICENSE-2.0
+#
+#       Unless required by applicable law or agreed to in writing, software
+#       distributed under the License is distributed on an "AS IS" BASIS,
+#       WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#       See the License for the specific language governing permissions and
+#       limitations under the License.
+#
+version: "2.4"
+services:
+
+  # AA Components
+  modelservice:
```

Comment:
i see one issue here for startup Compose does not wait until a container is “ready”  only until it’s running.
please see https://docs.docker.com/compose/startup-order/

Reply:
I am using the docker compose 2.4 features "healthcheck" + "depends_on condition" to ensure containers start up when their dependencies are ready. Containers will not start until their dependencies report success on their healthcheck command.

Reply:
I am fine with this for now, but we need to be able to move forward as Docker Compose releases new versions. In their docs they mention a `wait-for.sh` script that we can use.

Reply:
I do have a version 3 one that does work, but it relies on the modelservice dying and restarting until it works (usually takes 2 times), so I wasn't happy with it. The wait for it script might be the best approach for a v3 compose script unless we refactor the model service to be more graceful about Kafka not being available.